components/script/dom/extendablemessageevent.rs

```diff
@@ -123,6 +129,21 @@ impl ExtendableMessageEvent {
         );
         Extendablemessageevent.upcast::<Event>().fire(target);
     }
+
+    pub fn dispatch_error(target: &EventTarget, scope: &GlobalScope) {
+        let init = ExtendableMessageEventBinding::ExtendableMessageEventInit::empty();
+        let ExtendableMsgEvent = ExtendableMessageEvent::new(
+            scope,
+            atom!("messageerror"),
+            init.parent.parent.bubbles,
+            init.parent.parent.cancelable,
+            init.data.handle(),
+            init.origin.clone().unwrap(),
+            init.lastEventId.clone().unwrap(),
+            Vec::<DomRoot<MessagePort>>::new(),
```

Comment:
init.ports will be available, if you uncomment https://github.com/servo/servo/blob/2b24cfed10e01d91dd44a1015cc553a0fc09bbab/components/script/dom/webidls/ExtendableMessageEvent.webidl#L23

Reply:
Okay, I'll uncomment this and replace the initialized MessagePort vec with the parents port property.